server/client/stress_test.go

```diff
@@ -188,34 +187,34 @@ func doTest(t *testing.T, flags []string, numEvents, numSpans, numFrames, concur
 }
 
 func TestSmallTransactionsSequential(t *testing.T) {
-	doTest(t, noFlags, "10", "10", "10", "1")
+	doTest(t, noFlags, "10", "10", "10", "10", "1")
 }
 
 func TestSmallTransactionsLowConcurrency(t *testing.T) {
-	doTest(t, noFlags, "10", "10", "10", "5")
+	doTest(t, noFlags, "10", "10", "10", "10", "5")
 }
 
 func TestMediumTransactionsSequential(t *testing.T) {
-	doTest(t, noFlags, "20", "20", "20", "1")
+	doTest(t, noFlags, "20", "20", "20", "20", "1")
 }
 
 func TestMediumTransactionsLowConcurrency(t *testing.T) {
-	doTest(t, noFlags, "20", "20", "20", "5")
+	doTest(t, noFlags, "20", "20", "20", "20", "5")
 }
 
 func TestLargeTransactionsSequential(t *testing.T) {
-	doTest(t, noFlags, "30", "30", "30", "1")
+	doTest(t, noFlags, "30", "30", "30", "30", "1")
 }
 
 func TestLargeTransactionsLowConcurrency(t *testing.T) {
-	doTest(t, noFlags, "30", "30", "30", "5")
+	doTest(t, noFlags, "30", "30", "30", "30", "5")
 }
 
 func TestLargeTransactionsLowConcurrencyCustomFlags(t *testing.T) {
 	flags := []string{"-E", "output.elasticsearch.bulk_max_size=5000", "-E", "queue.mem.events=5000", "-E", "apm-server.concurrent_requests=10"}
-	doTest(t, flags, "30", "30", "30", "5")
+	doTest(t, flags, "30", "30", "30", "30", "5")
 }
 
 func TestErrorsVeryHighConcurrency(t *testing.T) {
-	doTest(t, noFlags, "10", "0", "100", "100")
+	doTest(t, noFlags, "10", "10", "0", "100", "100")
 }
```

Comment:
stress tests failed with a timeout on my machine.

Reply:
are you passing the env vars? 
https://github.com/elastic/hey-apm/blob/master/README.md#testing

Reply:
ah, that is the default Golang timeout. i added a note here:
https://github.com/elastic/hey-apm/blob/master/server/client/stress_test.go#L162

but you only see that if you run tests with the verbose flag `go test -v`, which is a bit unfortunate...

Reply:
Tests succeeded with higher timeout, thanks @jalvz